packages/effects-core/src/plugins/animation-graph/node-asset-type.ts

```
@@ -0,0 +1,17 @@
+import type { GraphNodeData } from '..';
+import type { Constructor } from '../../utils';
+
+const nodeDataClassStore: Record<string, any> = {};
+
+export function nodeDataClass (className: string) {
+  return (target: Object, context?: unknown) => {
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)